projects/go-style-guide/src/app/features/ui-kit/components/progress-bar-docs/progress-bar-docs.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-progress-bar-docs',
+  templateUrl: './progress-bar-docs.component.html',
+})
+export class ProgressBarDocsComponent implements OnInit {
+  pageTitle: string = 'Progress bar';
```

Comment:
Similar to my comment to the html piece of this component above, should these strings be translated?

Reply:
we don’t t have translations on for goponents